tools/astarte_e2e/lib/astarte_e2e/data_trigger.ex

```diff
@@ -0,0 +1,86 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule AstarteE2E.DataTrigger do
+  require Logger
+
+  use Task
+
+  alias AstarteE2E.Config
+
+  def start_link(opts) do
+    Task.start_link(__MODULE__, :install_data_trigger!, [opts])
+  end
+
+  def install_data_trigger!(opts) do
+    device_id = Keyword.fetch!(opts, :device_id)
+    base_url = Config.realm_management_url!()
+    realm = Config.realm!()
+    astarte_jwt = Config.jwt!()
+
+    url = "#{base_url}/v1/#{realm}/triggers"
```

Comment:
Let's use [`Path.join`](https://hexdocs.pm/elixir/1.12/Path.html#join/1), which also handles trailing slashes in path tokens

```suggestion
    url = Path.join([base_url, "v1", realm, "triggers"])
```